src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -56,46 +56,44 @@
         emoji                 (reagent/atom (emoji-picker.utils/random-emoji))
         account-name          (reagent/atom "")
         on-change-text        #(reset! account-name %)
-        show-account-origin   #(rf/dispatch [:show-bottom-sheet
-                                             {:content account-origin/view}])]
+        show-account-origin   #(rf/dispatch [:show-bottom-sheet {:content account-origin/view}])
+        derivation-path       (reagent/atom "")
+        set-derivation-path   #(reset! derivation-path %)]
     (fn [{:keys [theme]}]
-      (let [number-of-accounts                    (count (rf/sub
-                                                          [:wallet/accounts-without-watched-accounts]))
-            {:keys [address customization-color]} (rf/sub [:profile/profile])
-            {:keys [new-keypair]}                 (rf/sub [:wallet/create-account])
-            keypairs                              (rf/sub [:wallet/keypairs])
-            selected-keypair-uid                  (rf/sub [:wallet/selected-keypair-uid])
-            placeholder                           (i18n/label :t/default-account-placeholder)
-            derivation-path                       (utils/get-derivation-path
-                                                   number-of-accounts)
-            keypair                               (some #(when (= (:key-uid %) selected-keypair-uid)
-                                                           %)
-                                                        keypairs)
-            primary-keypair?                      (= selected-keypair-uid (:key-uid (first keypairs)))
```

Comment:
These calculations are completely based on values returned by subscriptions, so I moved them to subs